Neos.Flow/Classes/Command/ResourceCommandController.php

```diff
@@ -275,8 +273,8 @@ public function cleanCommand()
                 }
             }
             $response = null;
-            while (!in_array($response, ['y', 'n', 'c'])) {
-                $response = $this->output->ask('<comment>Do you want to remove all broken resource objects and related assets from the database? (y/n/c) </comment>');
```

Comment:
it seems the `cancel` option was meant to be a hard stop while `n` would ask for each image.
This was NEVER implement that way not even for the first draft robert did. Thats why i removed `cancel` as its equivalent to `no`.